### PR TITLE
Feature/us1496 every other week refactor

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: Gateway/crds-angular.test/Services/ServeServiceTest.cs [hunk 1]
```diff
@@ -666,43 +689,204 @@ public void RespondToServeOpportunityNoForEveryOtherWeek()
...
+            //    new Event
+            //    {
+            //        EventId = 4, EventStartDate = new DateTime(2015,1,22)
+            //    },
+            //    new Event
+            //    {
+            //        EventId = 5, EventStartDate = new DateTime(2015,1,29)
+            //    },
+            //};
+
```
Comment (IngageBrewerM) [inline Gateway/crds-angular.test/Services/ServeServiceTest.cs]: Remove commented out code

File: Gateway/crds-angular/Services/ServeService.cs [hunk 2]
```diff
@@ -259,23 +262,31 @@ public Capacity OpportunityCapacity(int opportunityId, int eventId, int? minNeed
...
+                        : HandleNoRsvp(participant, @event, opportunityIds, token);
+
+                    if (response.ToNullableObject<Opportunity>("previousOpportunity") != null)
+                        previousOpportunity = response.ToNullableObject<Opportunity>("previousOpportunity");
+
+                    templateId = (templateId != AppSetting("RsvpChangeTemplate"))
+                        ? response.ToInt("templateId")
+                        : templateId;
+
+                    sequenceDate = sequenceDate.AddDays(increment);
```
Comment (IngageBrewerM) [inline Gateway/crds-angular/Services/ServeService.cs]: Consider refactoring and pull this logic into well named methods that describe the logic better

**Comment by IngageMaddoxT** [comment]
@IngageBrewerM please review again

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**